simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt

```diff
@@ -292,7 +292,7 @@ class LocationFragment : Fragment(), LocationListener {
     val address: Address?
 
     try {
-      address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults)[0]
+      address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults)!![0]
```

Comment:
both these force unwraps are contained in generic `try {} catch (e: Exception) {}` blocks so it's 'safe' to do.

Reply:
No biggie, just aiming for readability. Good to merge 👏